.changeset/popular-vans-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/view-components': minor
```

Comment:
For now we're doing all `patch` I would like to have a conversation thought when we should start versioning properly

```suggestion
'@primer/view-components': patch
```

Reply:
Ah ok thanks!